ksonnet-gen/ksonnet/emit.go

```diff
@@ -68,6 +70,16 @@ func (root *root) emit(m *indentWriter) {
 		group.emit(m)
 	}
 
+	m.writeLine("local hidden = {")
```

Comment:
I completely agree, and I also thought about this, but I wasn't sure how good of an idea it would be to special-case `__hidden` (or whatever), so I went the conservative route.